Dockerfile.slim

```diff
@@ -16,16 +16,17 @@ RUN wget https://storage.googleapis.com/kubernetes-release/release/stable.txt &&
   chmod +x kubectl && \
   mv ./kubectl /usr/local/bin/kubectl
 
-RUN wget https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-06-05/bin/linux/amd64/heptio-authenticator-aws && \
-  chmod +x ./heptio-authenticator-aws && \
-  mv ./heptio-authenticator-aws /usr/local/bin/heptio-authenticator-aws
+RUN wget https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/linux/amd64/aws-iam-authenticator && \
+  chmod +x ./aws-iam-authenticator && \
+  mv ./aws-iam-authenticator /usr/local/bin/aws-iam-authenticator && \
+  ln -sf /usr/local/bin/aws-iam-authenticator /usr/local/bin/heptio-authenticator-aws
 
 RUN apk -v --update add py-pip && \
   pip install --upgrade awscli==1.14.5 s3cmd==2.0.1 python-magic && \
   apk -v --purge del py-pip && \
   rm /var/cache/apk/*
 
-ENV PATH "$PATH:/usr/local/bin/heptio-authenticator-aws"
+ENV PATH "$PATH:/usr/local/bin/aws-iam-authenticator"
```

Comment:
I think you only need to add /usr/local/bin to the path and can drop the aws-iam-authenticator portion here

Reply:
@edwinavalos, cool done!